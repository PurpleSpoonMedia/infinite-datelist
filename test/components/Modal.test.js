import { shallowMount } from '@vue/test-utils';
import Modal from '../../src/components/Modal.vue';

describe('Modal.vue', () => {
  test('should render Modal correctly', () => {
    const wrapper = shallowMount(Modal);
    expect(wrapper.html()).toMatchSnapshot();
  });
})
