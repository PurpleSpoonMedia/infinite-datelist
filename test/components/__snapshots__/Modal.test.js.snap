// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal.vue should render Modal correctly 1`] = `
<div class="modal modal-mask" style="display: block;" name="fade">
  <div role="document" class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header"></div>
      <div class="modal-body"></div>
      <div class="modal-footer"></div>
    </div>
  </div>
</div>
`;
